build-constraints.yaml

```diff
@@ -5486,40 +5486,66 @@ packages:
     # See "Large scale enabling/disabling of packages" in CURATORS.md for how to manage this section.
     #
     # This section is meant for libraries and executables that have
-    # bene disabled due to bounds issues, there is a separate section
+    # been disabled due to bounds issues, there is a separate section
     # for compilation failures as we need to build those packages to
     # verify if they have been fixeq.
     "Library and exe bounds failures":
+        - ALUT < 0 # tried ALUT-2.4.0.3, but its *library* requires the disabled package: OpenAL
+        - Agda < 0 # tried Agda-2.6.2.1, but its *library* does not support: bytestring-0.11.3.0
```

Comment:
Looks like we need to release Agda again:
- https://github.com/agda/agda/issues/5831